accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,475 @@
+    - `.ctor(InstanceContext, String)`
+    - `.ctor(InstanceContext, String, EndpointAddress)`
+* **DuplexClientBase**
+    - `.ctor(InstanceContext)`
+    - `.ctor(InstanceContext, String)`
+    - `.ctor(InstanceContext, String, EndpointAddress)`
+    - `.ctor(InstanceContext, String, String)`
+* **EndpointAddress**
+    - `ApplyTo(Message)`
+* **EndpointIdentity**
+    - `Equals(Object)`
+    - `GetHashCode()`
+* **EndpointNotFoundException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **FaultException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **InvalidMessageContractException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **MessageHeaderException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **MessageSecurityOverTcp**
+    - `get_ClientCredentialType()`
+    - `set_ClientCredentialType(MessageCredentialType)`
+* **NetHttpBinding**
+    - `CreateBindingElements()`
+* **NetHttpsBinding**
+    - `.ctor()`
+    - `.ctor(BasicHttpsSecurityMode)`
+    - `CreateBindingElements()`
+* **NetTcpBinding**
+    - `.ctor(String)`
+    - `CreateBindingElements()`
+* **ProtocolException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **QuotaExceededException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **ServerTooBusyException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **ServiceActivationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+
+### System.ServiceModel.Channels
+
+* **AddressHeader**
+    - `Equals(Object)`
+    - `GetHashCode()`
+* **MessageHeaders**
+    - `set_To(Uri)`
+* **SecurityBindingElement**
+    - `CreateSecureConversationBindingElement(SecurityBindingElement)`
+* **TransportSecurityBindingElement**
+    - `GetProperty<T>(BindingContext)`
+* **WebSocketTransportSettings**
+    - `get_DisablePayloadMasking()`
+    - `set_DisablePayloadMasking(Boolean)`
+
+### System.ServiceModel.Security
+
+* **MessageSecurityException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **SecurityAccessDeniedException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **SecurityNegotiationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **X509ServiceCertificateAuthentication**
+    - `set_CertificateValidationMode(X509CertificateValidationMode)`
+
+### System.Threading
+
+* **EventWaitHandle**
+    - `.ctor(Boolean, EventResetMode, String)`
+    - `.ctor(Boolean, EventResetMode, String, Boolean)`
+    - `OpenExisting(String)`
+    - `TryOpenExisting(String, EventWaitHandle)`
+* **Semaphore**
+    - `.ctor(Int32, Int32, String)`
+    - `.ctor(Int32, Int32, String, Boolean)`
+    - `OpenExisting(String)`
+    - `TryOpenExisting(String, Semaphore)`
+* **Thread**
+    - `SetApartmentState(ApartmentState)`
+* **WaitHandle**
+    - `SignalAndWait(WaitHandle, WaitHandle)`
+    - `SignalAndWait(WaitHandle, WaitHandle, Int32, Boolean)`
+    - `SignalAndWait(WaitHandle, WaitHandle, TimeSpan, Boolean)`
```

Comment:
why are these Windows only?

Reply:
@kouvel, do these not work on unix?

Reply:
They should work on Unix too, I think they were fixed to work since 2.1 with https://github.com/dotnet/coreclr/pull/16383